File: CodeKata/Loop - 3/Python/70. One Character Difference.py
v, p = input().split()

if len(v) != len(p):
    print("no")
else:
    diff = 0
    for a, b in zip(v, p):
        if a != b:
            diff += 1

    if diff == 1:
        print("yes")
    else:
        print("no")

File: CodeKata/String - (196)/JavaScript/70. One Character Difference.js
const readline = require("readline");

const inp = readline.createInterface({
    input: process.stdin
});

const userInput = [];

inp.on("line", (data) => {
    userInput.push(data);
});

inp.on("close", () => {

    let vp = userInput[0].split(" ")
    let v = vp[0]
    let p = vp[1]

    if (v.length !== p.length) {
        console.log("no");
    } else {
        let diff = 0;
        for (let i = 0; i < v.length; i++) {
            if (v[i] !== p[i]) {
                diff++;
            }
        }
        if (diff === 1) {
            console.log("yes");
        } else {
            console.log("no");
        }
    }

});